prep-javascript-dom-events/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Events</title>
+</head>
+
+<body >
+  <button class="hello-world-btn">Hello World!</button>
+  <script>
+document.addEventListener('DOMContentLoaded', function () {
+  console.log('content loaded!');
+});
+
+window.addEventListener('mousemove', function () {
+  console.log('mouse is moving!');
+});
+
+// eslint-disable-next-line no-unused-vars
+var $helloWorld = document.querySelector('Click', function () {
+  console.log('hello world');
+});
```

Comment:
Hi Louisa, for this part of the assignment, the console should have hello world printed each time you click the button.

Reply:
<img width="971" alt="Screen Shot 2022-03-05 at 2 04 48 PM" src="https://user-images.githubusercontent.com/99847558/156901472-2b633a7d-bae7-47d6-8ac8-18312c4c79d6.png">